sa/database.go

```diff
@@ -182,3 +183,10 @@ func initTables(dbMap *gorp.DbMap) {
 	dbMap.AddTableWithName(core.SignedCertificateTimestamp{}, "sctReceipts").SetKeys(true, "ID").SetVersionCol("LockCol")
 	dbMap.AddTableWithName(core.FQDNSet{}, "fqdnSets").SetKeys(true, "ID")
 }
+
+func safeSelectOne(m *gorp.DbMap, holder interface{}, q string, args ...interface{}) error {
+	if strings.ContainsAny(q, "'\"\n\r\\") {
```

Comment:
Ok. That all sounds reasonable. :+1: Thanks for linking to the `mysql_real_escape_string` implementation.